modules/ROOT/pages/push/proc_importing-and-exporting-device-metadata.adoc

```diff
@@ -0,0 +1,44 @@
+include::{partialsdir}/attributes.adoc[]
```

Comment:
@jhellar when you get a chance, can you review this new procedure too please - thanks, John